pom.xml

```diff
@@ -277,6 +286,12 @@
                     </configuration>
                 </plugin>
 
+                <plugin>
+                  <groupId>org.apache.maven.plugins</groupId>
+                  <artifactId>maven-resources-plugin</artifactId>
+                  <version>3.1.0</version>
```

Comment:
Shouldn't that reference the version property?

Reply:
Good catch, thanks!